torchci/rockset/metrics/__sql/force_merge_red_percent.sql

```diff
@@ -0,0 +1,29 @@
+WITH all_merges AS (
+  SELECT
+    _event_time as time,
+    skip_mandatory_checks,
+    LENGTH(failed_checks) AS failed_checks_count,
+  FROM
+    commons.merges
+  WHERE
+    _event_time >= PARSE_DATETIME_ISO8601(:startTime)
+    AND _event_time < PARSE_DATETIME_ISO8601(:stopTime)
+    AND owner = :owner
+    AND project = :project
+),
+force_merges_with_failed_checks AS (
+  SELECT
+    time,
+    IF(skip_mandatory_checks = true AND failed_checks_count > 0, 1, 0) AS force_merges_red,
+  FROM
+    all_merges
+)
+SELECT
+  FORMAT_TIMESTAMP('%Y-%m-%d', DATE_TRUNC(:granularity, time)) AS granularity_bucket,
```

Comment:
Might be worth converting this to a 2 week moving average to de-noise the signal

Reply:
Will do, but I need to be careful here as this query is very expensive, so I need to avoid making it run slower